brion/blueConfig.h

```diff
@@ -74,6 +74,12 @@ class BlueConfig : public boost::noncopyable
                                       const std::string& sectionName,
                                       const std::string& key ) const;
 
+    /**
+     * @return the user and circuit file target parsers.
+     * @version 1.7
+     */
+    BRION_API Targets getTargetParsers() const;
```

Comment:
Just call _config.parseTarget() in brain::Circuit and and I don't see a need for this method?